README.md

```diff
@@ -14,6 +14,34 @@ See the [documentation for the Maven Plugin](https://github.com/STAMP-project/bo
 
 ### Command line interface
 
+#### botsing preprocessing
+
+The latest version of Botsing command line (botsing-preprocessing-X-X-X.jar) is available at [https://github.com/STAMP-project/botsing/releases](https://github.com/STAMP-project/botsing/releases). 
+
+Botsing preprocessing has these mandatory parameters (key/value):
+ - `-i` represents the input file path (`crash_log`) with the stack trace to clean. For example `-i=path-name-of-crash-log`
+ - `-o` represents the output file path (`output_log`) cleaned of the error message and/or nested exceptions. For example `-o=path-name-of-output-log` 
+ 
+The actions to perform (clean) in the input log file are: 
+ - `-f` to flatten the stack trace. This action needs to use `-p` parameter (key/value) to set the package has frames pointing to the software under test (as **regexp**). For example `-p=my.package.*`. 
+ - `-e` to remove the error message. For example `-e`. 
+ 
+
+#### Example
+
+To clean the nested stack trace:
+
+```
+java -jar botsing-preprocessing.jar -i=crash_log.txt -o=output_log.log -f -p=com.example.*
+```
+
+or to remove the error message in the log file
+
+```
+java -jar botsing-preprocessing.jar -e -i=crash_log.txt -o==output_log.log 
```

Comment:
Could you double check this works with the latest version?

Reply:
Hi Xavier, I checked the preprocessing; I used this command (for example):
`java -jar botsing-preprocessing-1.0.5-SNAPSHOT-jar-with-dependencies.jar -i=nested_st2.txt -o=output_log.log -f -p=example.BusinessLevelAPI.businessLevelMethod` and it worked

the nested_st2.txt log is in the resources folder.
Anyway, I found again many compilation errors (_Do not use Windows line endings_) and I solved them simply using '**convert line to delimeter to -> unix**' and now I've all files to commit. I'm using eclipse Mars on win7.

Reply:
It worked also:
`java -jar botsing-preprocessing-1.0.5-SNAPSHOT-jar-with-dependencies.jar -i nested_st2.txt -o output_log.log -f -p example.BusinessLevelAPI.businessLevelMethod`